cmake/Config.cmake.in

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Pretty strange that this is here, is this an accident?

Reply:
probably my IDE doing some whitespace shenanigans.

Reply:
No, I mean the whole license. The code already has a license, there isn't really a need to have another, nor lists of authors, nor companies.